frame/staking/src/lib.rs

```diff
@@ -347,6 +348,10 @@ type NegativeImbalanceOf<T> = <<T as Config>::Currency as Currency<
 	<T as frame_system::Config>::AccountId,
 >>::NegativeImbalance;
 
+parameter_types! {
+	pub MaxUnlockingChunks: u32 = 32;
+}
```

Comment:
Why is this hardcoded and not a `Get`?

Reply:
This was the most direct conversion of the logic because we had it hardcoded here as a `const u32`, but we can add to the config